lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -112,6 +170,27 @@ def create_thread(request, course_id, commentable_id):
 
     thread.save()
 
+    follow = post.get('auto_subscribe', 'false').lower() == 'true'
+
+    if follow:
+        user = cc.User.from_django_user(request.user)
+        user.follow(thread)
```

Comment:
it seems these lines are duplicated later in this method on L174 - perhaps the later one should be removed?